futurenet/core/etc/stellar-core.cfg

```diff
@@ -9,6 +9,8 @@ UNSAFE_QUORUM=true
 FAILURE_SAFETY=0
 CATCHUP_RECENT=100
 
+EXPERIMENTAL_BUCKETLIST_DB=true
```

Comment:
Change to `DEPRECATED_SQL_LEDGER_STATE = false`

Reply:
We can only use the new flag if cfg files if all versions of stellar-core across the three image types support that new flag.

Reply:
I would prefer to do the update for all the configs simultaneously; we've already messed quickstart configs up quite a bit (because we haven't enabled BL DB on them), and it would be easier to do a simultaneous switch after pubnet has been updated.